### DISCUSSION TITLE
Why my sidebar is on the top of the page?

### DISCUSSION THREAD

**Discussion by ruanqizhen** [facebook/docusaurus#6133] [Q&A]
I create a new site by just running command `npx create-docusaurus@latest website classic`.

I can see the new site with `yarn start`, but it has some layout problems, the major one is that the sidebar is not on the side, but on the top of the page:  
![image](https://user-images.githubusercontent.com/4590897/146694590-1b11d83d-eb72-49ff-b8c8-22560b909a51.png)

Any idea how to fix this?

**Comment by Josh-Cena** [comment]
Hi, in case it's a compatibility problem, could you give your browser version? It seems some style sheets were not loaded (my guess)—have you tried reloading / restarting the dev server?

**Reply by ruanqizhen** [reply]
My browser is Chrome Version 96.0.4664.110 

I reads some demo pages, they all look good.  Only the new project I created I has this problem.  I'll try this with another browser.

**Reply by slorber** [reply]
Please
- submit your project as a github repro that we can inspect
- Check your browser devtools, for console errors and network errors
- re-generate a new project and see if this still happens

**Reply by ruanqizhen** [reply]
My other browsers can see the same problem.  This problem only happens when using `yarn start` to preview the pages.
After building it and reading from the build folder, everything looks just good.
I put my project in this repro:  https://github.com/ruanqizhen/docusaurus_test  
I'm on a Win10 system.

**Reply by slorber** [reply]
I can't reproduce this problem on my mac: 
![image](https://user-images.githubusercontent.com/749374/146906246-063690ff-a79d-436d-a9ec-3bf780f615d3.png)

Which other browsers exactly? Do you have screenshots to share?

Are there any terminal console logs?

Please
- Open the devtools console tabs, look for errors in the console
- Open the devtools network tab, look for errors when loading stylesheets/css
- Try to disable global/browser plugins, antivirus, intrusive technologies
- Try to use another computer session without all the above enabled
- Try to run in safe mode?
- Try on another computer

**Reply by slorber** [reply]
Can you inspect the CSS of the "skip to main content" button? Here's mine:

![image](https://user-images.githubusercontent.com/749374/146907081-f2c2055b-e9d7-4af8-b6a4-c84ca190fb90.png)

**Reply by ruanqizhen** [reply]
This is mine:
![image](https://user-images.githubusercontent.com/4590897/146935959-08d59f9f-e9e4-4f1f-8651-64b5a38787b4.png)

**Reply by ruanqizhen** [reply]
I found the problem.  There is some special characters (space, parentheses) in my project path.  After moving my project to a simple folder like "c:\projects", everything looks good.

**Reply by slorber** [reply]
Oh I see thanks for letting us know. 
We have seen similar issues in the past for some windows users.
Can you please give us the path you used before so that we can reproduce this issue more consistently? Eventually add a test for it

**Reply by ruanqizhen** [reply]
Yes, this was my project path: `C:\Users\ruanqizhen\Dropbox (Personal)\Documents_qq\github\`

**Reply by slorber** [reply]
Thanks for those details.

I can't reproduce on macOS, so it's probably related to your env.
Have you tried using another terminal (afaik there are many options for Windows).

If a contributor is willing to handle this issue, let me know. This seems related to how CSS module classnames are generated, cf the screenshots above. We use css-loader with `localIdentName: '[local]_[path][name]'` (removing `[path]` probably works, but do we want to?)

IMHO this issue could totally be related to the usage of Dropbox. Please move the project outside of dropbox, but keep a space in the name to be sure if it's related or not. Not sure it's a good idea to put your projects in Dropbox, considering node_modules would end up being uploaded to your dropbox account.
